src/main/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessage.java

```diff
@@ -1,19 +1,23 @@
 package co.rsk.federate.signing.hsm.message;
 
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+
 import org.ethereum.core.BlockHeader;
 import org.spongycastle.util.encoders.Hex;
 
 public class AdvanceBlockchainMessage {
     private final List<String> blockHeaders;
+    private final List<String[]> brothersList;
 
-    public AdvanceBlockchainMessage(List<BlockHeader> blockHeaders) {
+    public AdvanceBlockchainMessage(List<BlockHeader> blockHeaders, List<List<BlockHeader>> brothersList) {
```

Comment:
```suggestion
    public AdvanceBlockchainMessage(List<BlockHeader> blockHeaders, List<List<BlockHeader>> brothers) {
```
Don't use the data type as the name of the variable

Reply:
Resolved